pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "hacksquad-bot"
 version = "0.1.0"
 description = ""
-authors = ["Arpan Pandey <arpanpandey.aps@gmail.com>"]
+authors = ["HellFire#7769 (718395193090375700)", "sravan#0001 (468386640155508737)", "Capt. Pred#0495 (669223041322057769)"]
```

Comment:
This doesn't work like this
The authors must be in this format
`authors = ["User 1 <user1@gmail.com>" , "User 2 <user2@gmail.com>"]

Reply:
Can I just have IDs in <>? I don't really want to share my mail

Reply:
You can just leave it empty. But please try to keep the format as intended by Poetry.

Reply:
Righto!